docs/explanation/core/configuration.md

```diff
@@ -27,7 +27,7 @@ like the file below, however, double check by inspecting the
 
 # The version of the CometBFT binary that created or
 # last modified the config file. Do not modify this.
-version = "0.39.0"
+version = "1.0.0-dev"
```

Comment:
```suggestion
version = "1.0.0"
```

Reply:
We can't set it to exactly "1.0.0" on `main`, as we risk to confuse it with the first final release of `v1.x`